templates/nunjucks/_base.njk.html

```diff
@@ -10,6 +9,31 @@
 {% endblock %}
 
 {% block header %}
+{% if environmentDisplay %}
+<style>
+.environmentDisplay {
+  position: fixed;
+  background-color: #f47738;
+  color: #fff;
+  top: 0;
+  left: 0;
+  padding: 5px 10px;
+  font-family: nta,Arial,sans-serif;
+  text-transform: uppercase;
+  transform-origin: top left;
+  transform: rotate(90deg);
+  margin-left: 30px;
+  margin-top: 0px;
+  height: 30px;
+  overflow: hidden;
+  font-size: 16px;
+  box-sizing: border-box;
+}
+</style>
+<div class="environmentDisplay">
```

Comment:
minor: our convention is hyphenation I think

Reply:
Sorted